src/components/Textarea/Textarea.tsx

```diff
@@ -0,0 +1,46 @@
+import { forwardRef, useState } from 'react';
+
+import { StyledTextarea, StyledContainer, StyledHelperText } from './Textarea.style';
+import { TextareaProps } from './Textarea.type';
+
+export const Textarea = forwardRef<HTMLTextAreaElement, TextareaProps>(
+  (
+    { width, height, maxLength, helperText, placeholder, disabled, error, onValueChange, ...props },
+    ref
+  ) => {
+    const [value, setValue] = useState('');
+    const [currentLength, setCurrentLength] = useState(value?.length || 0);
+
+    const handleChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+      const newValue = e.target.value;
+      if (maxLength && e.target.value.length > maxLength) {
```

Comment:
이왕 `newValue` 선언한 거 여기도 쓸 수 있지 않을까요!!

```suggestion
      if (maxLength && newValue.length > maxLength) {
```

Reply:
아주 좋습니다!!!!!!!!!! 짱